readthedocs/core/middleware.py

```diff
@@ -34,8 +34,7 @@ def process_request(self, request):
         path = request.get_full_path()
         log_kwargs = dict(host=host, path=path)
         public_domain = getattr(settings, 'PUBLIC_DOMAIN', None)
-        production_domain = getattr(settings, 'PRODUCTION_DOMAIN',
-                                    'readthedocs.org')
+        production_domain = getattr(settings, 'PRODUCTION_DOMAIN', 'readthedocs.org')
```

Comment:
This line looks to have been previously passing lint @ 80 chars, I'd say aggressively linted lines should trump less linted lines. We are currently checking for 100 character length, but talked about dropping to 80 eventually.